react/Trigger.tsx

```diff
@@ -1,60 +1,92 @@
+import {
+  useResponsiveValue,
+  MaybeResponsiveInput,
+} from 'vtex.responsive-values'
 import classnames from 'classnames'
 import { useCssHandles } from 'vtex.css-handles'
 import React, { useEffect, useRef } from 'react'
 
 import {
+  usePopoverState,
   usePopoverDispatch,
   PopoverContextProvider,
 } from './components/PopoverContext'
 
 const CSS_HANDLES = ['trigger']
 
-// TODO hover trigger
-type TriggerMode = 'click' | 'hover'
+export type TriggerMode = 'click' | 'hover'
 
 interface Props {
   children: React.ReactNode
-  trigger?: TriggerMode
+  trigger?: MaybeResponsiveInput<TriggerMode>
```

Comment:
I see why using responsive-values could be useful, but I also think that you should not allow for `'hover'` on mobile